draft/spec/change-log.md

```diff
@@ -0,0 +1,108 @@
+---
+no_site_title: true
+---
+<img src="https://avatars0.githubusercontent.com/u/35607965" alt="OCFL Hand-drive logo" style="float:right;width:307px;height:307px;"/>
+# Oxford Common File Layout Specification v1.1 Change Log
+{:.no_toc}
+
+Draft 6 September 2022
+
+**Editors:**
+
+* [Neil Jefferies](https://orcid.org/0000-0003-3311-3741), [Bodleian Libraries, University of Oxford](http://www.bodleian.ox.ac.uk/)
+* [Rosalyn Metz](https://orcid.org/0000-0003-3526-2230), [Emory University](https://web.library.emory.edu/)
+* [Julian Morley](https://orcid.org/0000-0003-4176-1933), [Stanford University](https://library.stanford.edu/)
+* [Simeon Warner](https://orcid.org/0000-0002-7970-7855), [Cornell University](https://www.library.cornell.edu/)
+* [Andrew Woods](https://orcid.org/0000-0002-8318-4225), [Harvard University](https://library.harvard.edu/)
+
+This document is licensed under a [Creative Commons Attribution 4.0
+License](https://creativecommons.org/licenses/by/4.0/). [OCFL logo:
+"hand-drive"](https://avatars0.githubusercontent.com/u/35607965) by
+[Patrick Hochstenbach](http://orcid.org/0000-0001-8390-6171) is
+licensed under [CC BY 2.0](https://creativecommons.org/licenses/by/2.0/).
+
+## Changes from OCFL v1.0 to v1.1
+
+[Version 1.1 of the OCFL Specification](https://ocfl.io/draft/spec/) is a [minor version](https://semver.org/) update to the [OCFL Specification v1.0](https://ocfl.io/1.0/spec/). It is intended to add functionality in a backwards compatible manner, and to clarify certain aspects based on a community feedback.
```

Comment:
Suggestion from:
```
, and to clarify certain aspects based on a community feedback
```
to:
```
by limiting changes to correction and clarification.
```

Reply:
I think we are adding one new thing which is the idea of prior version conformance so it wouldn't be true to say strictly corrections and clarification.  How about:

> The focus is correction and clarification, plus the addition of backwards compatible rules for the conformance of prior object versions and the associated specification version number sequence.

Reply:
Looks good.

Reply:
Fixed in [f53c4f6](https://github.com/OCFL/spec/pull/610/commits/f53c4f6055fed95dd6838eca8d1a9877f93c224a)